### DISCUSSION TITLE
Futbol

### DISCUSSION THREAD

**Discussion by KayraMertAydin** [sonsuzus/sonsuzus#516] [Soru Cevap]
def a(n, gol):
    skor = {}
    for go in gol:
        if go in skor:
            skor[go] += 1
        else:
            skor[go] = 1
    b = max(skor, key=skor.get)
    return b
n = int(input())
gol = [input().strip() for _ in range(n)]
print(a(n, gol))
#

**Comment by sonsuzus** [comment]
Soru nerede? 
Ayrıca kodunu code  bloğuna almalısın her zaman.

**Comment by KayraMertAydin** [comment]
Bir gün Vasya, Berland 1910 Futbol Şampiyonası'nın final maçının sonuçlarına bakmaya karar verdi. Ne yazık ki maçın genel skorunu bulamadı; ancak, maçın sürecinin derin bir açıklamasına ulaştı. Açıklamanın toplamda **n** satırı vardı ve her bir satır bir golü tanımlıyordu. Her gol, golü atan takımın adıyla işaretlenmişti. Vasya'ya yardımcı olun, finali kazanan takımın adını öğrenmesine yardımcı olun. Maçın berabere bitmediği garanti edilmektedir.

**Girdi**  
İlk satırda bir tamsayı **n** (1 ≤ n ≤ 100) — açıklamadaki satır sayısı verilmiştir. Ardından, her gol için sırasıyla n satır gelir — her satır, golü atan takımın adını içerir. Takımların isimleri, uzunlukları 10 harften fazla olmayan, büyük harflerden oluşan boş olmayan satırlardır. Maçın berabere bitmediği ve açıklamanın en fazla iki farklı takımı içerdiği garanti edilmektedir.

**Çıktı**  
Kazanan takımın adını yazdırın. Unutmayın, futbolda daha fazla gol atan takım kazanan olarak kabul edilir.

```
def a(n, gol):
    skor = {}
    for go in gol:
        if go in skor:
            skor[go] += 1
        else:
            skor[go] = 1
    b = max(skor, key=skor.get)
    return b
n = int(input())
gol = [input().strip() for _ in range(n)]
print(a(n, gol))

```
